content/docs/intro/quickstart.md

```diff
@@ -69,7 +69,7 @@ Released smiling-penguin
 
 In the example above, the `stable/mysql` chart was released, and the name of our
 new release is `smiling-penguin`. You get a simple idea of the features of this
-MySQL chart by running `helm inspect stable/mysql`.
+MySQL chart by running `helm inspect chart stable/mysql`.
```

Comment:
Good catch. Might as well put the proper helm v3 format which is `helm show all stable/mysql`

Reply:
I've updated to `helm show chart` and added a short description for `helm show all`